extensions/amp-animation/0.1/parse-animation-config.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {childElementByTag} from '../../../src/dom';
+import {tryParseJson} from '../../../src/json';
+import {user, userAssert} from '../../../src/log';
+
+/**
+ * @param {!Element} element
+ * @return {?JsonObject}
+ */
+export function parseAnimationConfig(element) {
```

Comment:
Would this helper work instead?

https://github.com/ampproject/amphtml/blob/9cc61e6bdd1b218aafe65c81dc382f3751797f38/src/json.js#L135-L157

Reply:
Nice!